types/tests.ts

```diff
@@ -70,8 +66,8 @@ module('rendering', function (hooks) {
     const inputFormat2 = this.get('inputFormat');
 
     // render the component on the page
-    this.render(hbs`<div>bar</div>`);
-    assert.equal(this.element.querySelector('div')?.innerText, 'bar');
+    await render(hbs`<div data-test-content>bar</div>`);
+    assert.equal('[data-test-content]', 'bar');
```

Comment:
📎 did you mean `assert.dom('[data-test-content]').hasText('bar')`?

Reply:
(it's just a types test, so 🤷 lol)

Reply:
You're right that this test isn't correct, but if I try that syntax, I get `Property 'dom' does not exist on type 'Assert'.` as the error, so I need to think about this some more.

Reply:
I suppose that makes sense -- we don't have qunit dom here. 

we could probably delete the line, tbh. 

`ember-qunit`'s scope is pretty small:
 - this.owner
 - setup*Test
 - start
 
 ... and I think that's it? so anything outside of those methods, is just _extra_ testing of things not in this library

Reply:
hmmm ok